src/components/new-safe/create/steps/AdvancedOptionsStep/index.tsx

```diff
@@ -183,15 +176,57 @@ const AdvancedOptionsStep = ({ onSubmit, onBack, data, setStep }: StepRenderProp
                 },
                 required: true,
               })}
+              fullWidth
               label="Salt nonce"
               error={Boolean(formState.errors[AdvancedOptionsFields.saltNonce]) || Boolean(isDeployed)}
               helperText={
-                (formState.errors[AdvancedOptionsFields.saltNonce]?.message ?? Boolean(isDeployed))
-                  ? 'The Safe is already deployed. Use a different salt nonce.'
-                  : undefined
+                formState.errors[AdvancedOptionsFields.saltNonce]?.message ??
+                (Boolean(isDeployed) ? 'The Safe is already deployed. Use a different salt nonce.' : undefined)
               }
             />
+
+            <Typography
+              variant="h5"
+              sx={{
+                fontWeight: 700,
+                display: 'inline-flex',
+                alignItems: 'center',
+                gap: 1,
+                mt: 4,
+                width: 1,
+              }}
+            >
+              Payment receiver
+              <Tooltip title="The payment receiver changes the predicted Safe address." arrow placement="top">
+                <span style={{ display: 'flex' }}>
+                  <SvgIcon component={InfoIcon} inheritViewBox color="border" fontSize="small" />
+                </span>
+              </Tooltip>
+            </Typography>
+            <Typography
+              variant="body2"
+              sx={{
+                mb: 2,
+              }}
+            >
+              Impacts the derived Safe address
+            </Typography>
+            <TextField
+              {...register(AdvancedOptionsFields.paymentReceiver, {
+                required: true,
+              })}
+              label="Payment receiver"
+              error={Boolean(formState.errors[AdvancedOptionsFields.paymentReceiver]) || Boolean(isDeployed)}
+              helperText={
+                formState.errors[AdvancedOptionsFields.paymentReceiver]?.message ??
+                (Boolean(isDeployed) ? 'The Safe is already deployed. Use a different payment receiver.' : undefined)
+              }
+              fullWidth
+            />
```

Comment:
Was that someone's requirement?

Reply:
Should we add a tip along the lines of "if you created your Safe before 09.2024, leave this empty"?